Razer device manager - NullPointerException

### Expected Behaviour
Razer device manager should have started but it never starts

### Actual Behaviour
The program runs as normaly do but when I go to Configuration/DeviceManager  the start button is enabled in Razer row

### Reproduction steps
Install Aurora from binary. Without any Razer software installed.
Go to Configuration/Device&Wrappers/Razer and install Chroma.
Restart PC

### Frequency
Every time. I have tested a lot of times restarting the PC.

### Aurora Version:
beta v0.8.1

### Previously an Issue?
Don't know. I have tested in other computer with the same keyboard and it works great

### LOGS
```
[ERROR] 18:16:36: Razer device, Exception! Message:System.NullReferenceException: Referencia a objeto no establecida como instancia de un objeto.
   en System.Object.GetType()
   en Corale.Colore.Core.RegistryHelper.IsSdkAvailable()
   en Aurora.Devices.Razer.RazerDevice.Initialize() en C:\projects\aurora\Project-Aurora\Project-Aurora\Devices\Razer\RazerDevice.cs:línea 60
[INFO] 18:16:36: Device, Razer, was not initialized
```

--- a/Project-Aurora/Project-Aurora/Profiles/LeagueOfLegends/GameEvent_LoL.cs
+++ b/Project-Aurora/Project-Aurora/Profiles/LeagueOfLegends/GameEvent_LoL.cs
@@ -20,7 +20,7 @@ namespace Aurora.Profiles.LeagueOfLegends
     {
         private const string URI = "https://127.0.0.1:2999/liveclientdata/allgamedata";
 
-        private readonly HttpClient client = new HttpClient();
+        private readonly HttpClient client;
         private readonly Timer updateTimer;
 
         private _RootGameData allGameData;
@@ -29,7 +29,9 @@ namespace Aurora.Profiles.LeagueOfLegends
         public GameEvent_LoL() : base()
         {
             //ignore ssl errors
-            ServicePointManager.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;
+            HttpClientHandler handler = new HttpClientHandler();
+            handler.ServerCertificateCustomValidationCallback = (message, cert, chain, sslPolicyErrors) => { return true; };
+            client = new HttpClient(handler);
             updateTimer = new Timer(100);
             updateTimer.Elapsed += UpdateData;
         }

